terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/fibo/1.pos.program'
./scripts/neg/ijcai19/fibo-val/fibo-NEGLITE-1-0-a-val.sh: line 1: 28639 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/fibo/domain.pddl domains/neg/fibo/fibo-5-neg-a.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/fibo/1.pos.program'
./scripts/neg/ijcai19/fibo-val/fibo-NEGLITE-1-0-a-val.sh: line 2: 28641 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/fibo/domain.pddl domains/neg/fibo/fibo-5-neg-b.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/fibo/1.pos.program'
./scripts/neg/ijcai19/fibo-val/fibo-NEGLITE-1-0-a-val.sh: line 3: 28643 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/fibo/domain.pddl domains/neg/fibo/fibo-5-neg-c.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/fibo/1.pos.program'
./scripts/neg/ijcai19/fibo-val/fibo-NEGLITE-1-0-a-val.sh: line 4: 28645 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/fibo/domain.pddl domains/neg/fibo/fibo-5-neg-d.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/fibo/1.pos.program'
./scripts/neg/ijcai19/fibo-val/fibo-NEGLITE-1-0-a-val.sh: line 5: 28647 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/fibo/domain.pddl domains/neg/fibo/fibo-5-pos-a.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/fibo/1.pos.program'
./scripts/neg/ijcai19/fibo-val/fibo-NEGLITE-1-0-a-val.sh: line 6: 28649 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/fibo/domain.pddl domains/neg/fibo/fibo-6-neg-a.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/fibo/1.pos.program'
./scripts/neg/ijcai19/fibo-val/fibo-NEGLITE-1-0-a-val.sh: line 7: 28651 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/fibo/domain.pddl domains/neg/fibo/fibo-6-neg-b.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/fibo/1.pos.program'
./scripts/neg/ijcai19/fibo-val/fibo-NEGLITE-1-0-a-val.sh: line 8: 28653 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/fibo/domain.pddl domains/neg/fibo/fibo-6-neg-c.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/fibo/1.pos.program'
./scripts/neg/ijcai19/fibo-val/fibo-NEGLITE-1-0-a-val.sh: line 9: 28655 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/fibo/domain.pddl domains/neg/fibo/fibo-6-neg-d.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/fibo/1.pos.program'
./scripts/neg/ijcai19/fibo-val/fibo-NEGLITE-1-0-a-val.sh: line 10: 28657 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/fibo/domain.pddl domains/neg/fibo/fibo-6-pos-a.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/fibo/1.pos.program'
./scripts/neg/ijcai19/fibo-val/fibo-NEGLITE-1-0-a-val.sh: line 11: 28659 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/fibo/domain.pddl domains/neg/fibo/fibo-7-neg-a.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/fibo/1.pos.program'
./scripts/neg/ijcai19/fibo-val/fibo-NEGLITE-1-0-a-val.sh: line 12: 28661 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/fibo/domain.pddl domains/neg/fibo/fibo-7-neg-b.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/fibo/1.pos.program'
./scripts/neg/ijcai19/fibo-val/fibo-NEGLITE-1-0-a-val.sh: line 13: 28663 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/fibo/domain.pddl domains/neg/fibo/fibo-7-neg-c.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/fibo/1.pos.program'
./scripts/neg/ijcai19/fibo-val/fibo-NEGLITE-1-0-a-val.sh: line 14: 28665 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/fibo/domain.pddl domains/neg/fibo/fibo-7-neg-d.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/fibo/1.pos.program'
./scripts/neg/ijcai19/fibo-val/fibo-NEGLITE-1-0-a-val.sh: line 15: 28667 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/fibo/domain.pddl domains/neg/fibo/fibo-7-pos-a.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/fibo/1.pos.program'
./scripts/neg/ijcai19/fibo-val/fibo-NEGLITE-1-0-a-val.sh: line 16: 28669 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/fibo/domain.pddl domains/neg/fibo/fibo-8-neg-a.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/fibo/1.pos.program'
./scripts/neg/ijcai19/fibo-val/fibo-NEGLITE-1-0-a-val.sh: line 17: 28671 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/fibo/domain.pddl domains/neg/fibo/fibo-8-neg-b.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/fibo/1.pos.program'
./scripts/neg/ijcai19/fibo-val/fibo-NEGLITE-1-0-a-val.sh: line 18: 28673 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/fibo/domain.pddl domains/neg/fibo/fibo-8-neg-c.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/fibo/1.pos.program'
./scripts/neg/ijcai19/fibo-val/fibo-NEGLITE-1-0-a-val.sh: line 19: 28675 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/fibo/domain.pddl domains/neg/fibo/fibo-8-neg-d.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/fibo/1.pos.program'
./scripts/neg/ijcai19/fibo-val/fibo-NEGLITE-1-0-a-val.sh: line 20: 28677 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/fibo/domain.pddl domains/neg/fibo/fibo-8-pos-a.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/fibo/1.pos.program'
./scripts/neg/ijcai19/fibo-val/fibo-NEGLITE-1-0-a-val.sh: line 21: 28679 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/fibo/domain.pddl domains/neg/fibo/fibo-9-neg-a.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/fibo/1.pos.program'
./scripts/neg/ijcai19/fibo-val/fibo-NEGLITE-1-0-a-val.sh: line 22: 28681 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/fibo/domain.pddl domains/neg/fibo/fibo-9-neg-b.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/fibo/1.pos.program'
./scripts/neg/ijcai19/fibo-val/fibo-NEGLITE-1-0-a-val.sh: line 23: 28683 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/fibo/domain.pddl domains/neg/fibo/fibo-9-neg-c.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/fibo/1.pos.program'
./scripts/neg/ijcai19/fibo-val/fibo-NEGLITE-1-0-a-val.sh: line 24: 28685 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/fibo/domain.pddl domains/neg/fibo/fibo-9-neg-d.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/fibo/1.pos.program'
./scripts/neg/ijcai19/fibo-val/fibo-NEGLITE-1-0-a-val.sh: line 25: 28687 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/fibo/domain.pddl domains/neg/fibo/fibo-9-pos-a.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/fibo/1.pos.program'
./scripts/neg/ijcai19/fibo-val/fibo-NEGLITE-1-0-a-val.sh: line 26: 28689 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/fibo/domain.pddl domains/neg/fibo/fibo-2-pos-a.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/fibo/1.pos.program'
./scripts/neg/ijcai19/fibo-val/fibo-NEGLITE-1-0-a-val.sh: line 27: 28691 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/fibo/domain.pddl domains/neg/fibo/fibo-3-pos-a.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/fibo/1.pos.program'
./scripts/neg/ijcai19/fibo-val/fibo-NEGLITE-1-0-a-val.sh: line 28: 28693 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/fibo/domain.pddl domains/neg/fibo/fibo-1-pos-a.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/fibo/1.pos.program'
./scripts/neg/ijcai19/fibo-val/fibo-NEGLITE-1-0-a-val.sh: line 29: 28695 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/fibo/domain.pddl domains/neg/fibo/fibo-1-neg-a.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/fibo/1.pos.program'
./scripts/neg/ijcai19/fibo-val/fibo-NEGLITE-1-0-a-val.sh: line 30: 28697 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/fibo/domain.pddl domains/neg/fibo/fibo-3-neg-b.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/fibo/1.pos.program'
./scripts/neg/ijcai19/fibo-val/fibo-NEGLITE-1-0-a-val.sh: line 31: 28699 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/fibo/domain.pddl domains/neg/fibo/fibo-2-neg-b.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/fibo/1.pos.program'
./scripts/neg/ijcai19/fibo-val/fibo-NEGLITE-1-0-a-val.sh: line 32: 28701 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/fibo/domain.pddl domains/neg/fibo/fibo-2-neg-c.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/fibo/1.pos.program'
./scripts/neg/ijcai19/fibo-val/fibo-NEGLITE-1-0-a-val.sh: line 33: 28703 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/fibo/domain.pddl domains/neg/fibo/fibo-2-neg-d.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/fibo/1.pos.program'
./scripts/neg/ijcai19/fibo-val/fibo-NEGLITE-1-0-a-val.sh: line 34: 28705 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/fibo/domain.pddl domains/neg/fibo/fibo-1-neg-c.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/fibo/1.pos.program'
./scripts/neg/ijcai19/fibo-val/fibo-NEGLITE-1-0-a-val.sh: line 35: 28707 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/fibo/domain.pddl domains/neg/fibo/fibo-4-neg-a.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/fibo/1.pos.program'
./scripts/neg/ijcai19/fibo-val/fibo-NEGLITE-1-0-a-val.sh: line 36: 28709 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/fibo/domain.pddl domains/neg/fibo/fibo-2-neg-a.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/fibo/1.pos.program'
./scripts/neg/ijcai19/fibo-val/fibo-NEGLITE-1-0-a-val.sh: line 37: 28711 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/fibo/domain.pddl domains/neg/fibo/fibo-4-neg-b.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/fibo/1.pos.program'
./scripts/neg/ijcai19/fibo-val/fibo-NEGLITE-1-0-a-val.sh: line 38: 28713 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/fibo/domain.pddl domains/neg/fibo/fibo-1-neg-b.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/fibo/1.pos.program'
./scripts/neg/ijcai19/fibo-val/fibo-NEGLITE-1-0-a-val.sh: line 39: 28715 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/fibo/domain.pddl domains/neg/fibo/fibo-4-neg-c.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/fibo/1.pos.program'
./scripts/neg/ijcai19/fibo-val/fibo-NEGLITE-1-0-a-val.sh: line 40: 28717 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/fibo/domain.pddl domains/neg/fibo/fibo-3-neg-c.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/fibo/1.pos.program'
./scripts/neg/ijcai19/fibo-val/fibo-NEGLITE-1-0-a-val.sh: line 41: 28719 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/fibo/domain.pddl domains/neg/fibo/fibo-1-neg-d.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/fibo/1.pos.program'
./scripts/neg/ijcai19/fibo-val/fibo-NEGLITE-1-0-a-val.sh: line 42: 28721 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/fibo/domain.pddl domains/neg/fibo/fibo-3-neg-a.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/fibo/1.pos.program'
./scripts/neg/ijcai19/fibo-val/fibo-NEGLITE-1-0-a-val.sh: line 43: 28723 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/fibo/domain.pddl domains/neg/fibo/fibo-3-neg-d.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/fibo/1.pos.program'
./scripts/neg/ijcai19/fibo-val/fibo-NEGLITE-1-0-a-val.sh: line 44: 28725 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/fibo/domain.pddl domains/neg/fibo/fibo-4-neg-d.pddl $1
